src/main/java/eu/darkbot/DarkBot.java

```diff
@@ -0,0 +1,9 @@
+package eu.darkbot;
+
+//main class.
+public class DarkBot {
+
+    public static void main(String[] args) {
+        //...
```

Comment:
:thinking:

Reply:
as main class in da future instead of Bot.java